Gemfile.lock

```diff
@@ -239,17 +234,23 @@ GEM
       rails (>= 3.0)
     globalid (0.3.7)
       activesupport (>= 4.1.0)
-    google-api-client (0.7.1)
-      addressable (>= 2.3.2)
-      autoparse (>= 0.3.3)
-      extlib (>= 0.9.15)
-      faraday (>= 0.9.0)
-      jwt (>= 0.1.5)
-      launchy (>= 2.1.1)
-      multi_json (>= 1.0.0)
-      retriable (>= 1.4)
-      signet (>= 0.5.0)
-      uuidtools (>= 2.1.0)
+    google-api-client (0.9.28)
```

Comment:
is there a reason to upgrade to google-api-client 0.9.28 from 0.7.1 ? because based on [changelog](https://github.com/google/google-api-ruby-client/blob/master/CHANGELOG.md#090pre1) the version 0.9 introduces breaking changes and we can check this guide for migration: [link](https://github.com/google/google-api-ruby-client/blob/master/MIGRATING.md#migrating-from-version-08x-to-09-or-above)